src/FixMessageUtil.h

```diff
@@ -114,6 +114,7 @@ class FixMessageUtil {
 						addFixGroups(group, entry);
 
 						map->addGroup(group->field(), *group);
+            delete group;
```

Comment:
I don't think this is safe to delete here...we can't delete the groups from the message as it has not been sent out yet. The quickfix library will then assume there isn't any groups on the message. Does the test-suite pass with this change? This deletion should be handled by the destructor for FIX::FieldMap